public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -40,12 +38,12 @@ include ../../../../_includes/_util-fns
   when the definion of `Engine` changes, our `Car` class must change.
   That makes `Car` brittle.
 
-  What if we want to put a different brand of tires on our `Car`. Too bad.
+  What if we want to put a different brand of tires on our `Car`? Too bad.
   We're locked into whatever brand the `Tires` class creates. That makes our `Car` inflexible.
 
   Right now each new car gets its own engine. It can't share an engine with other cars.
   While that makes sense for an automobile engine,
-  we can think of other dependencies that should be shared ... like the onboard
+  we can think of other dependencies that should be shared... like the onboard
```

Comment:
Maybe we should go with neither ellipsis nor dash but rather the full, unbroken sentence?
See my comment.